tracer/build/_build/Honeypot/IntegrationGroups.cs

```diff
@@ -127,11 +131,24 @@ private async Task PopulatePackages()
                 var latestPackage = potentiallySupportedPackages.First();
                 var latestVersion = new Version(latestPackage.Identity.Version.ToNormalizedString());
 
+                var latestSupportedPackage = potentiallySupportedPackages
+                    .FirstOrDefault(x => x.Identity.Version.Version <= MaximumAssemblyVersion);
```

Comment:
Any worry about assembly version not matching nuget package version?

Reply:
Yeah, it's a concern, and Oracle.ManagedDataAccess has completely different version numbers, so in that case I fall back to what we had before. But either way, it's better than what we had before, and looks ok having eye-balled it

Reply:
I only realised today that this file contains a mapping between assembly names and NuGet packages. That's actually pretty useful, means we should be able to generate 90% of the package generator json file programmatically in the future, so we don't accidentally miss testing integrations that we add support for (*cough* Npgsql 5.0.0 *cough*)